docs/contributing/contributing.md

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
```suggestion
---
order: 1
description: <add description>
parent:
  order: 9
  title: Contributing Guide
---
```

We need to follow a proper order, description and title method as per requirements and same is applicable on other files also :)

Reply:
good catch @soniasingla I've removed front matter from the file since we aren't publishing this file, following the same example as the core contributing.md file https://github.com/tendermint/starport/edit/develop/contributing.md